tasks/activate_account_user.py

```diff
@@ -12,6 +13,7 @@
 class ActivateAccountUserTask(BaseCodecovTask, name=activate_account_user_task_name):
     def run_impl(
         self,
+        _db_session: Session,
```

Comment:
Potentially dumb question, but does adding a session in any way affect any transaction related things? Like will the BaseCodecovTask use the session in any way that could silently interfere with anything within the task?

Reply:
Not a dumb question! I don't think it will affect the transaction because the session is created but isn't used. So there's nothing in an uncommitted transaction to cause issues. AFAICT, the ProcessFlakesTask _also_ creates a session without using it, and there hasn't been any trouble with Django DB access.

Reply:
Cool cool, yeah that makes sense, even if it were to commit something in the parent class, there's nothing in this task itself that's using the session itself